frontend/src/components/tracks/index-pm.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import Header from './header/index';
+import Footer from '../shared/footer/index';
+import TrackImage from './track-image/index';
+import WhatYouLearn from './what-you-learn/index';
+import OfferResponsibilities from './offer-responsibilities/index';
+import Expectations from './expectations/index';
+import { or, header, expectations } from './data/pm';
+
+/**
+ * The Apply Now button is not abstracted away in its own component, because it's temporary there
+ * TrackImage is for now the same for all tracks. It can be later made more flexible if track heads
+ * decide to use different images.
+ */
+class TrackPM extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Header data={header} logo="/assets/tracks/dps-logo-white.png" />
```

Comment:
Would be nice to also having the track as parameter here. As we need to customize the styling a little bit per track (e.g. colors). 
Does it make sense to put it here?

Reply:
Another option would be to duplicate this stuff too. 🤔

Reply:
I was thinking about passing className, in this case, the component is agnostic and the parent takes control, but for even more customization a parameter makes more sense. I think both (param or className) are fine 👍